src/clj_momo/lib/es/index.clj

```diff
@@ -19,6 +19,29 @@
   "make a template uri from a host and a template name"
   (format "%s/_template/%s" uri template-name))
 
+(s/defn rollover-uri :- s/Str
+  "make a rollover uri from a host and an index name"
+  ([uri alias] (rollover-uri uri alias nil false))
+  ([uri :- s/Str
+    alias :- s/Str
+    new-index-name :- (s/maybe s/Str)
+    dry_run :- s/Bool]
+   (str (index-uri uri alias)
+        "/_rollover"
+        (when new-index-name
+          (str "/" new-index-name))
+        (when dry_run
+          "?dry_run"))))
```

Comment:
Instead of several when, I often prefer to use `cond->` like this:

```clojure
(cond-> (str (index-uri uri alias) "/_rollover")
  new-index-name (str "/" new-index-name)
  dry_run (str "?dry_run"))
```

Reply:
exactly